src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -1114,32 +1114,32 @@ public function providePathsForMakePathRelative()
             ['/aa/bb/cc', '/aa/bb/cc/dd/', '../'],
             ['/aa/bb/cc/', '/aa/bb/cc/dd', '../'],
             ['/aa/bb/cc/', '/aa/bb/cc/dd/', '../'],
-            ['/aa/bb/cc', '/aa', 'bb/cc/'],
```

Comment:
Please don't delete previous tests. 
Only add new cases

Reply:
It changed not deleted. Please see my explanation in the description. Without changing the expectation, we can't fix the method.

Reply:
indeed, I misread the description.